src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/UserDefinedImplicitConversions.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.CodeAnalysis.CSharp
     internal abstract partial class ConversionsBase
     {
         /// <remarks>
-        /// NOTE: Keep this method in sync with AnalyzeImplicitUserDefinedConversionForSwitchGoverningType.
+        /// NOTE: Keep this method in sync with AnalyzeImplicitUserDefinedConversionForV6SwitchGoverningType.
```

Comment:
Consider using standard cref syntax so that go to definition and rename work for this reference.